test/commands/expire.spec.js

```diff
@@ -24,7 +24,7 @@ describe("The 'expire' method", function () {
                 client.EXPIRE(["expiry key", "1"], helper.isNumber(1));
                 setTimeout(function () {
                     client.exists(["expiry key"], helper.isNumber(0, done));
-                }, 1100);
+                }, 3000);
```

Comment:
Was this an issue on windows? Redis should never take that long to execute any command :/